src/main/daml/Daml/Finance/Instrument/Swap/Fpml/Util.daml

```diff
@@ -502,6 +503,7 @@ calculateFloatingPaymentClaimsFromSwapStream floatingRateCalculation s
         [] -> 0.0
         [se] -> se.initialValue
         _ -> error "Multiple SpreadSchedules not yet supported"
+      compoundedRefRate = floatingRateCalculation.floatingRateIndex == "USD-SOFR-COMPOUND"
```

Comment:
I would move this logic into a separate utility function, as this is likely to be extended when further currencies are supported